Tasks/AzureIoTEdgeV2/deployimage.ts

```diff
@@ -43,18 +43,42 @@ class azureclitask {
 
       this.loginAzure();
 
-      tl.debug('OS release:' + os.release());
-
+      tl.debug('OS release:' + os.release());      
       let showIotExtensionCommand = ["extension", "show", "--name", "azure-iot"];
       let result = tl.execSync('az', showIotExtensionCommand, Constants.execSyncSilentOption);
       if (result.code !== 0) { // The extension is not installed
+        let checkAzureIoTVersionExtensionCommand = ["--version"]; // temporary solution until the paho-mqtt situation goes away 
         let installFixedIotExtensionCommand = ["extension", "add", "--source", Constants.azureCliIotExtensionDefaultSource, "-y", "--debug"];
         let installLatestIotExtensionCommand = ["extension", "add", "--name", "azure-iot", "--debug"];
         try {
+          let outputStream: EchoStream = new EchoStream();
```

Comment:
the setuptools upgrade is extra logic needed by the az cli, i suggest we move it into installAzureCliIotExtension function